scripts/harvestToContours.py

```diff
@@ -35,7 +35,7 @@
 parser.add_argument("--interpolation",   type = str, help="interpolation function for scipy (RBF): linear, cubic, gaussian, multiquadric or (griddata): nearest, linear, cubic", default = "multiquadric")
 parser.add_argument("--interpolationScheme",   type = str, help="type of interpolation for scipy: rbf, griddata", default = "rbf")
 parser.add_argument("--interpolationEpsilon", type=float, help="scipy (RBF) epsilon parameter", default = 0)
-parser.add_argument("--level",           type = float, help="contour level output. Default to 95%% CL", default = 1.64485362695)
+parser.add_argument("--level",           type = float, help="contour level output. Default to 95% CL. Note: 1-sided.", default = 1.64485362695)
```

Comment:
```suggestion
parser.add_argument("--level",           type = float, help="contour level output. Default to one-sided 95% CL.", default = 1.64485362695)
```

Reply:
I think is probably makes more sense to put it right smack dab in the there so that someone doesn't miss this while glancing at it. It is also more common in my experience to see people explicitly mention the type of confidence interval (one-sided vs. two-sided) that the confidence level is referring to.